glean-core/csharp/Glean/Metrics/StringListMetricType.cs

```diff
@@ -0,0 +1,172 @@
+﻿using Mozilla.Glean.FFI;
+using System;
+using System.Text.Json;
+
+namespace Mozilla.Glean.Private
+{
+    /// <summary>
+    /// This implements the developer facing API for recording string list metrics.
+    ///
+    /// Instances of this class type are automatically generated by the parsers at build time,
+    /// allowing developers to record values that were previously registered in the metrics.yaml file.
+    ///
+    /// The string list API only exposes the [add] and [set] methods, which takes care of validating the input
+    /// data and making sure that limits are enforced.
+    ///
```

Comment:
```suggestion
```

I know these are available in the Android docs, but the are outdated anyway. Let's not carry them over.